app/backend/src/couchers/tasks.py

```diff
@@ -139,16 +139,24 @@ def send_email_changed_notification_email(user):
     email.enqueue_email_from_template(user.email, "email_changed_notification", template_args={"user": user})
 
 
-def send_email_changed_confirmation_email(user, token, expiry_text):
+def send_email_changed_confirmation_email(user, token, expiry_text, old):
```

Comment:
Let's have one of these `send_*_email` functions per template, so divide this into two functions